impl_core/src/main/java/io/opencensus/implcore/stats/RecordUtils.java

```diff
@@ -83,15 +97,34 @@
     for (int i = 0; i < columns.size(); ++i) {
       TagKey tagKey = columns.get(i);
       if (!tags.containsKey(tagKey)) {
-        // replace not found key values by null.
-        tagValues.add(UNKNOWN_TAG_VALUE);
+        @javax.annotation.Nullable TagValue tagValue = UNKNOWN_TAG_VALUE;
+        if (RPC_TAG_MAPPINGS.containsKey(tagKey)) {
```

Comment:
nit: this is a double lookup of the key.  It would be better to just get() it, and check for Null

Reply:
Done, good catch.